pom.xml

```diff
@@ -14,6 +14,10 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <!-- Dependency versions -->
+    <sonarVersion>3.3.0.603</sonarVersion>
+    <jacocoVersion>0.7.8</jacocoVersion>
```

Comment:
Keep this consistent (namings and versions) with the Platform root POM (waiting for NXBT-87).
Upgrading versions on master is even better, while both are equals.

Reply:
Good idea, but let's do it in another ticket. I'll create one for that.